website/src/components/UI/Affix.tsx

```diff
@@ -0,0 +1,99 @@
+import type { CSSProperties, ReactNode, FC } from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+import { getDomStyle } from '../../utils';
+
+interface Props {
+  style: CSSProperties;
+  children: ReactNode;
+}
+
+const AffixContent = styled.div`
+`;
+
+const getPositionStyle = (
+  hasCssSticky: boolean,
+  defaultStyles: CSSProperties = {},
+):CSSProperties => {
+  const { width } = defaultStyles;
+  const positionStyle: CSSProperties = hasCssSticky ? {
+    position: 'sticky',
+    marginLeft: `-${width}px`,
+    display: 'inline-block',
+    float: 'left',
+  } : {
+    position: 'absolute',
+  };
+
+  return {
+    ...positionStyle,
+    ...defaultStyles,
+  };
+};
+
+const Affix: FC<Props> = (props) => {
```

Comment:
just use `React.FC` type, no need `Props`

Reply:
```ts
interface Props {
  style: CSSProperties;
  children: ReactNode;
}
```

the props has customized `style` key,  i can't immediate use it

Reply:
You can remove the `children` prop.

![image](https://user-images.githubusercontent.com/48400568/169726620-8755f6cc-e10a-4b9b-ac98-fee2861e5d65.png)